libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -26,7 +27,7 @@ const AP_Param::GroupInfo AP_RPM::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: RPM type
     // @Description: What type of RPM sensor is connected
-    // @Values: 0:None,1:PWM,2:AUXPIN,3:EFI,4:Harmonic Notch
+    // @Values: 0:None,1:PWM,2:AUXPIN,3:EFI,4:Harmonic Notch,5:ESC Telemetry
```

Comment:
maybe add type 6 as "average of vtol motor RPM" and type 7 as "average of fwd motors"?